### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by cgeroldinger** [punxaphil/custom-sonos-card#44] [Show and tell]
Hi Johan

Not sure where the best place is to say thank you for all your effort on that.
Me and my wife really like the sonos-card a lot and use it neaerly every day on our Living room ipad.  It gets better and better.
![image](https://user-images.githubusercontent.com/40886256/157655925-02d78c5d-d504-473a-8c8e-f00a36626b4b.png)

Thanks

**Comment by punxaphil** [comment]
Nice to hear!
[I like coffee](https://www.buymeacoffee.com/punxaphil) 🤗

**Reply by cgeroldinger** [reply]
Coffee....Good oint.  Sent....:-)

**Reply by punxaphil** [reply]
☕️🙂
